controller/tap/server.go

```diff
@@ -105,8 +107,16 @@ func (s *server) TapByResource(req *public.TapByResourceRequest, stream pb.Tap_T
 	}
 
 	for _, pod := range pods {
+		// create the expected pod identity from the pod spec
+		name := fmt.Sprintf("%s.%s.serviceaccount.identity.%s.cluster.local", pod.Spec.ServiceAccountName, req.Target.Resource.Namespace, s.controllerNamespace)
+		log.Debugf("requiring pod name={%s}", name)
```

Comment:
a bit more context in this log line would be helpful.  something like "initiating tap request to %s with required id %s"